src/main/java/com/google/api/codegen/php/PhpGapicContext.java

```diff
@@ -132,8 +132,10 @@ public String addImport(String typeName) {
    * Returns the PHP representation of a reference to a type.
    */
   public String typeName(TypeRef type) {
-    if (type.isMap() || type.isRepeated()) {
+    if (type.isMap()) {
       return "array";
+    } else if (type.isRepeated()) {
+      return basicTypeName(type) + "[]";
```

Comment:
This makes sense but I am concerned that this is not consistent with the current php client library